Havit.Blazor.Components.Web.Bootstrap/NamedViews/HxNamedViewList.razor.cs

```diff
@@ -1,13 +1,14 @@
 ﻿namespace Havit.Blazor.Components.Web.Bootstrap;
 
 /// <summary>
-/// List of named views for <see cref="HxListLayout{TFilterModel}" />.<br />
-/// Full documentation and demos: <see href="https://havit.blazor.eu/components/HxNamedViewList">https://havit.blazor.eu/components/HxNamedViewList</see>
+/// List of named-views for <see cref="HxListLayout{TFilterModel}" />.<br />
+/// Full documentation and demos: <see href="https://havit.blazor.eu/components/HxListLayout#named-views">https://havit.blazor.eu/components/HxListLayout#named-views</see>
 /// </summary>
 
 public partial class HxNamedViewList<TFilterModel>
 {
 	[Parameter] public IEnumerable<NamedView<TFilterModel>> NamedViews { get; set; }
+	[Parameter] public NamedView<TFilterModel> SelectedNamedView { get; set; }
```

Comment:
We cannot have `SelectedNameView` and `OnNamedViewSelected` parameters as their form a pair for data-binding a thus we have to introduce a breaking-change and consolidate the pair to:
```
[Parameter] public NamedView<TFilterModel> SelectedNamedView { get; set; }
[Parameter] public EventCallback<NamedView<TFilterModel>> SelectedNamedViewChanged { get; set; }
```

The original `OnNamedViewSelected` has to be marked as `[Obsolete]` and will be removed in the future.